build.d/800-permissions

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 set -e
+
```

Comment:
```suggestion
```

Reply:
an adaption of 800-permissions is no longer needed nor done